--- src/Configuration/DefaultConfigPass.php	2020-06-02 20:52:13.866133964 +0000
+++ src/Configuration/DefaultConfigPass.php	2020-06-02 20:54:46.969861233 +0000
@@ -55,7 +55,7 @@
     {
         $defaultMenuItem = $this->findDefaultMenuItem($backendConfig['design']['menu']);

-        if ('empty' === $defaultMenuItem['type']) {
+        if (is_array($defaultMenuItem) && 'empty' === $defaultMenuItem['type']) {
             throw new \RuntimeException(sprintf('The "menu" configuration sets "%s" as the default item, which is not possible because its type is "empty" and it cannot redirect to a valid URL.', $defaultMenuItem['label']));
         }

